src/psbt.h

```diff
@@ -1322,12 +1316,6 @@ struct PSBTOutput
             if (m_asset.IsNull() && m_asset_commitment.IsNull()) {
                 throw std::ios_base::failure("Output asset is required in PSET");
             }
-            if (!m_value_commitment.IsNull() && amount != nullopt && m_blind_value_proof.empty()) {
-                throw std::ios_base::failure("Blind value proof must be provided if explicit value and commitment are provided");
-            }
-            if (!m_asset_commitment.IsNull() && !m_asset.IsNull() && m_blind_asset_proof.empty()) {
-                throw std::ios_base::failure("Blind asset proof must be provided if explicit asset and commitment are provided");
-            }
```

Comment:
In 0bb738f73712022516db75802adcc4bd74ddb19b "test: add decodepsbt and analyzepsbt tests for blinding proofs"

The removal of these checks seems to be better suited for an earlier commit.